exonum/src/node/connect_list.rs

```diff
@@ -0,0 +1,33 @@
+use blockchain::ValidatorKeys;
+use crypto::PublicKey;
+use helpers::fabric::NodePublicConfig;
+use std::collections::HashMap;
+use std::net::SocketAddr;
+
+/// Mapping between peers and public keys
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct ConnectList {
+    /// Peers
+    pub peers: HashMap<SocketAddr, PublicKey>,
+}
+
+impl ConnectList {
+    /// Create ConnectList from validators_keys and peers
+    pub fn from_keys(validators_keys: &Vec<ValidatorKeys>, peers: &Vec<SocketAddr>) -> Self {
+        let peers: HashMap<SocketAddr, PublicKey> = peers
+            .iter()
+            .zip(validators_keys.iter())
+            .map(|(p, v)| (*p, v.consensus_key))
+            .collect();
+
+        ConnectList { peers }
+    }
+
+    /// Create ConnectList from NodePublicConfigs
```

Comment:
This information is easily retrieved from method signature. I think you should avoid docstrings of that kind.
What about "Creates `ConnectList` from validators public configs"?